.github/workflows/test-script.yaml

```diff
@@ -0,0 +1,12 @@
+name: test-example-scripts
```

Comment:
Can you drop this? I don't really care to have tests for the helper script since it's just provided as a convenience.

Reply:
Yep. Done.